.github/workflows/plugin-build.yml

```diff
@@ -0,0 +1,207 @@
+          composer install --no-scripts --no-dev
+          composer dump-autoload
+
+          # Build the core plugin
+          # When the 'npx grunt build' is done running it creates a build folder in the elementor folder
+          # and in the /tmp/elementor-builds. These builds are the same
+          npx grunt build
+
+          CORE_PACKAGE_VERSION=$(node -p "require('./package.json').version")
+          echo "CORE_PACKAGE_VERSION=${CORE_PACKAGE_VERSION}" >> $GITHUB_ENV
+
+          cd /tmp/elementor-builds/
+          mv ./${CORE_PACKAGE_VERSION} ./elementor
+          zip -r elementor.zip ./elementor
+
+          cd $ROOT_DIR
+
+          mv /tmp/elementor-builds/elementor.zip ./elementor.zip
+
+      - name: Build Plugin 2
+        id: build_plugin
+        run: |
+          npm config set git-tag-version false
+          export PUPPETEER_SKIP_DOWNLOAD=true
+
+          npx grunt build
+
+      - name: Zip Elementor Build Files
+        id: zip_elementor
+        if: ${{ env.REPOSITORY_NAME == 'elementor' }}
+        run: |
+          PLUGIN_FILENAME=elementor-${{ env.BUILD_PACKAGE_VERSION }}
+          PLUGIN_ZIP_FILENAME=${PLUGIN_FILENAME}.zip
+
+          mv build elementor
+          zip -r ${PLUGIN_ZIP_FILENAME} elementor
+
+          echo "PLUGIN_FILENAME=${PLUGIN_FILENAME}" >> $GITHUB_ENV
+
+
+      - name: Zip Elementor Pro Build Files
+        id: zip_elementor_pro
+        if: ${{ env.REPOSITORY_NAME == 'elementor-pro' }}
+        run: |
+          ROOT_DIR=$(pwd)
+          PLUGIN_FILENAME="elementor-pro-${{ env.BUILD_PACKAGE_VERSION }}"
+          PLUGIN_ZIP_FILENAME="${PLUGIN_FILENAME}.zip"
+
+          PACKAGE_BASE_DIR="/tmp/elementor-pro-releases/"
+          PACKAGE_PATH=$(ls $PACKAGE_BASE_DIR)
+          PLUGIN_ZIP_PATH=$(pwd)/${PLUGIN_ZIP_FILENAME}
+
+          # When the 'npx grunt build' is done running in PRO it creates a zip build file in
+          # /tmp/elementor-pro-releases folder named elementor-pro-{PACKAGE_VERSION}.zip
+          # So we move the zip generated to the root folder of the repository
+          mv "${PACKAGE_BASE_DIR}/${PACKAGE_PATH}" ${PLUGIN_ZIP_FILENAME}
+
+          # Unzip the zip file in the root folder into the /tmp folder
+          unzip -q ${PLUGIN_ZIP_PATH} -d /tmp/
+
+          # Remove the zip file from the root folder
+          rm ${PLUGIN_ZIP_PATH}
+
+          cd /tmp/
+          zip -r -q ${PLUGIN_ZIP_PATH} ./elementor-pro
+          cd $ROOT_DIR
+
+          unzip -qo ${PLUGIN_ZIP_PATH} -d ./elementor-pro
+
+          echo "PLUGIN_ZIP_FILENAME=${PLUGIN_ZIP_FILENAME}" >> $GITHUB_ENV
+          echo "PLUGIN_FILENAME=${PLUGIN_FILENAME}" >> $GITHUB_ENV
+
+      - name: Publish zip on push to release or develop
+        shell: bash
+        if: env.REPOSITORY_NAME == 'elementor-pro' && github.event_name == 'push' && startsWith(github.repository, 'elementor/') && (startsWith(github.ref, 'refs/heads/release/') || github.ref == 'refs/heads/develop')
```

Comment:
@davseve  is this step needed let me know I am not understanding cause the endpoint is passed so i dont know where its going. ``` php       - name: Publish zip on push to release or develop
        shell: bash
        if: env.REPOSITORY_NAME == 'elementor-pro' && github.event_name == 'push' && startsWith(github.repository, 'elementor/') && (startsWith(github.ref, 'refs/heads/release/') || github.ref == 'refs/heads/develop')
        run: curl --fail -F "package=@${{ env.PLUGIN_ZIP_FILENAME }}" "${{ secrets.DEPLOY_BUILDS_ENDPOINT }}&type=pro"
 ```

Reply:
I think that this step is not valid anymore
In the past all the release branches were under the release prefix E.g, 'release/3.20.0`.
As well we had a develop branch, 
Every release used to get merged to the develop branch so that it always used to include the last updated release. 
@louiswol94 WDYT?
Do we still need this step?

Reply:
I don't think we need it.

Reply:
@Batsirai-muchareva 
Than this condition can be deleted

Reply:
@davseve removed